pydis_site/apps/content/resources/guides/python-guides/windows/microsoft-store.md

```diff
@@ -1,36 +1,73 @@
-  which isn't added to PATH on the Microsoft Store version of Python.
-
-* ##### It can cause issues with permissions
-  Some modules and scripts wont work with it because of restricted permissions.
-  This is explained
-  [in the Python documentation](https://docs.python.org/3/using/windows.html#redirection-of-local-data-registry-and-temporary-paths)
-
-* ##### It uses PATH inconsistently
-  `python.exe` isn't added to PATH, but `pip.exe` is. This makes it easy to have
-  an inconsistent `pip` and `python`. The `pip` is under the
-  `C:\Users\username\AppData\Local\Microsoft\WindowsApps` entry, which wouldn't
-  obviously have anything Python related in it. If you add your normal Python
-  install path below this one, it's `pip` will be overridden by the `WindowsApps`
-  one but the `python` one wont as it's overwriting the app execution alias.
-
-* ##### Confusing App Execution Alias Behaviour
-  The usage of app execution aliases is confusing. If you use the online
-  installer and for get to tick the box to to add to path, when you try and use
-  `python`, you either get no output or the Microsoft Store opens. This hides the
-  command not found error which would have indicated that their install didn't
-  work as expected.
+-   ##### Command line tools won't work
+
+    Most command line tools, like [`black`](https://pypi.org/project/black/) or
+    [`pyinstaller`](https://pypi.org/project/pyinstaller/), won't work directly (without specifying the full path, or
+    invoking them as a module if they allow it). This happens because they normally work by adding an executable to the
+    `\Scripts` directory, which isn't added to Path on the Microsoft Store version of Python.
+
+-   ##### It can cause issues with permissions
+
+    Some modules and scripts wont work with it because of restricted permissions. This is explained [in the Python
+    documentation](https://docs.python.org/3/using/windows.html#redirection-of-local-data-registry-and-temporary-paths)
+
+-   ##### It can cause Path confusion
+
+    Path is the Windows environment variable that determine what programs run when you type commands in terminals.
+    (Actually there are two Paths, a user one and a system one that has precedence, but they get [combined when
+    used](https://superuser.com/a/878382/935845).) With Python from the Microsoft Store installed, your `python`, `py`,
+    and `pip` commands may get mixed up with the python.org ones, depending on the order and contents of Path.
+
+    The Path entry for the Microsoft Store is `C:\Users\<user>\AppData\Local\Microsoft\WindowsApps`. The entries for the
+    full python.org install are commonly `C:\Users\<user>\AppData\Local\Programs\Python\Python311` and
+    `C:\Users\<user>\AppData\Local\Programs\Python\Python311\Scripts` (for Python 3.11). If for some reason you want
+    both installed, put the ones you want the commands for higher up in Path.
```

Comment:
Hmm, I see a `python.exe` in my `C:\Users\<user>\AppData\Local\Microsoft\WindowsApps`. When the store version is installed that exe runs python, and there's a pip.exe in there too, but when it's not installed it opens the store - aka it is the app execution alias. So I've been assuming it gets looked up on the path normally. If you toggle off the app execution alias that `python.exe` is removed.

So I think it's ok. I double checked and with `C:\Users\<user>\AppData\Local\Microsoft\WindowsApps` higher in path both `pip` and `python` run the store version.

Reply:
Interesting, I get a bunch of things but not `python.exe` so assumed it's just because that was left to the alias:
![image](https://user-images.githubusercontent.com/22353562/201528972-ebf3684e-2eac-403f-8d41-9ca6e003283a.png)
does it depend on the version?

Reply:
oh sorry I misread what you wrote, I understand now